example-library/src/main/java/uk/gov/gchq/palisade/example/config/ExamplePolicyPrepopulationFactory.java

```diff
@@ -16,69 +16,58 @@
 
 package uk.gov.gchq.palisade.example.config;
 
-import org.apache.avro.reflect.MapEntry;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.boot.context.properties.ConfigurationProperties;
 
 import uk.gov.gchq.palisade.Generated;
 import uk.gov.gchq.palisade.example.hrdatagenerator.types.Employee;
-import uk.gov.gchq.palisade.example.util.ExampleFileUtil;
-import uk.gov.gchq.palisade.resource.ParentResource;
 import uk.gov.gchq.palisade.resource.Resource;
-import uk.gov.gchq.palisade.resource.impl.DirectoryResource;
-import uk.gov.gchq.palisade.resource.impl.FileResource;
-import uk.gov.gchq.palisade.resource.impl.SystemResource;
 import uk.gov.gchq.palisade.rule.Rule;
-import uk.gov.gchq.palisade.service.PolicyCacheWarmerFactory;
-import uk.gov.gchq.palisade.service.UserCacheWarmerFactory;
+import uk.gov.gchq.palisade.service.PolicyPrepopulationFactory;
+import uk.gov.gchq.palisade.service.UserPrepopulationFactory;
 import uk.gov.gchq.palisade.service.request.Policy;
+import uk.gov.gchq.palisade.util.ResourceBuilder;
 
-import java.net.URI;
-import java.nio.file.FileSystems;
-import java.nio.file.Path;
-import java.nio.file.Paths;
+import java.io.File;
+import java.util.AbstractMap.SimpleImmutableEntry;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.Objects;
 import java.util.StringJoiner;
-import java.util.stream.StreamSupport;
 
-import static java.util.Objects.isNull;
 import static java.util.Objects.requireNonNull;
 
-@ConfigurationProperties
-public class ExamplePolicyCacheWarmerFactory implements PolicyCacheWarmerFactory {
+public class ExamplePolicyPrepopulationFactory implements PolicyPrepopulationFactory {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(ExamplePolicyCacheWarmerFactory.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExamplePolicyPrepopulationFactory.class);
 
     private String type = "";
-    private String resource = "";
+    private File resource;
```

Comment:
Why is this now a file? I thought these were all Strings because that is what Spring can easily pull from the config json